client/src/components/Admin/Orders/OrderDetailsLayout.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
see if we can use a common layout and reuse it everywhere. Avoid creating many layouts. Anyway, the code inside is the same for all layouts.